src/Augurk.CommandLine/Commands/PublishCommand.cs

```diff
@@ -279,6 +279,8 @@ private Feature ParseFeatureFile(string featureFile)
 
         private string ProcessDescription(string originalDescription)
         {
+            if (originalDescription == null) { return originalDescription; }
```

Comment:
Great catch! Personally I usually prefer the use of String.IsNullOrEmpty for performance considerations. You might even go for String.IsNullOrWhiteSpace, although that would imply further knowledge about the code in the deeper methods.